modules/configuring-network-subnets.adoc

```diff
@@ -33,12 +33,12 @@ $ cd new-install
 +
 ----
 networking:
-      clusterNetworks:
+      clusterNetwork:
         - cidr: 10.128.0.0/14
-          hostSubnetLength: 9
+          hostPrefix: 9
       machineCIDR: 10.0.0.0/16
-      serviceCIDR: 172.30.0.0/16
-     type: OpenshiftSDN
+      serviceNetwork: 172.30.0.0/16
+     networkType: OpenshiftSDN
```

Comment:
One more space indent (should be at same depth as `clusterNetwork`, `machineCIDR` and `serviceNetwork`)

Reply:
@sjug Fixed!